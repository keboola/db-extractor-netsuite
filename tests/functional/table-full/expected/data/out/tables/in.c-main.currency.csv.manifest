{"destination":"in.c-main.currency","incremental":false,"metadata":[{"key":"KBC.name","value":"Currency"},{"key":"KBC.sanitizedName","value":"Currency"},{"key":"KBC.schema","value":"default"},{"key":"KBC.type","value":"TABLE"}],"column_metadata":{"InternalId":[{"key":"KBC.datatype.type","value":"nvarchar"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"InternalId"},{"key":"KBC.sanitizedName","value":"InternalId"},{"key":"KBC.primaryKey","value":true}],"CurrencyPrecision":[{"key":"KBC.datatype.type","value":"nvarchar"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"CurrencyPrecision"},{"key":"KBC.sanitizedName","value":"CurrencyPrecision"},{"key":"KBC.primaryKey","value":false}],"DisplaySymbol":[{"key":"KBC.datatype.type","value":"nvarchar"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"DisplaySymbol"},{"key":"KBC.sanitizedName","value":"DisplaySymbol"},{"key":"KBC.primaryKey","value":false}]},"columns":["InternalId","CurrencyPrecision","DisplaySymbol"],"primary_key":["InternalId"]}
