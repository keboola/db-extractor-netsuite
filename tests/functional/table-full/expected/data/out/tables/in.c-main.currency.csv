"1","_two","$"
"2","_two","£"
"3","_two","$"
"4","_two","€"
"5","_two","$"
"6","_two","$"
"7","_two","R$"
"8","_two","R"
"9","_two","kr"
"10","_two","kr"
"11","_two","kr"
"12","_two","sFr."
"13","_two","lei"
